runtime/queries/jsx/textobjects.scm

```diff
@@ -1,3 +1,9 @@
 ; See runtime/queries/ecma/README.md for more info.
 
 ; inherits: _jsx,_javascript,ecma
+
+(jsx_self_closing_element) @xml_element.around @xml_element.inside
+
+(jsx_element (jsx_opening_element) (_)* @xml_element.inside (jsx_closing_element))
+
+(jsx_element) @xml_element.around
```

Comment:
Can you add this to `runtime/queries/_jsx/textobjects.scm` instead so that both `jsx` and `tsx` can inherit it? The file doesn't exist yet, but I tried it locally and it seems to work.

Reply:
Good catch, I've moved it